src/scenes/Partners/Edit/EditOrganizationName.graphql

```diff
@@ -1,12 +1,19 @@
-mutation UpdateOrganizationName($id: ID!, $name: String!) {
-  updateOrganization(input: { organization: { id: $id, name: $name } }) {
+mutation UpdateOrganization($id: ID!, $name: String!, $acronym: String) {
+  updateOrganization(
+    input: { organization: { id: $id, name: $name, acronym: $acronym } }
+  ) {
     organization {
       id
       name {
         canRead
         canEdit
         value
       }
+      acronym {
+        canRead
+        canEdit
+        value
+      }
       avatarLetters
```

Comment:
I'd like to use a fragment for organization to dedupe the fields between this mutation and the query.

It could go below `PartnerDetails`